### ISSUE TITLE
Clean up unclaimed folder, then delete dev

### ISSUE THREAD

**Issue by kurzum** [dbpedia/links#6]
- remove from master
- in dev try to move as many as possible into the links folder

**Comment by RPOD** [comment]
List of unclaimed links is [here](https://docs.google.com/spreadsheets/d/13GDNyU8m_UHhjEz4EvAJVMCXPKo_moXQM_4hWf9gdNU/edit#gid=0) also if they are reachable or not plus the contact data (if given). Should I contact people about their links and should I remove the unreachable links from unclaimed and move the reachable ones into the `dbpedia.org` links folder?

**Comment by kurzum** [comment]
Yes, please contact them. For the other ones could you send an email to DBpedia discussion?

**Comment by kurzum** [comment]
reposting:

```
remove from master
in dev try to move as many as possible into the links folder
```

**Comment by kurzum** [comment]
hm, do you know how to deal with all the files in unclaimed? this is not an easy task

**Comment by RPOD** [comment]
I try to generate as much meta-data for each linkset that is still reachable and create a simple README describing what the dataset is describing. If I run into issues I will ask right away.

**Comment by RPOD** [comment]
With the latest commits the several new linksets have been added to `dbpedia.org`, the `metadata.ttl` for each linkset has been updated to follow the conventions. 

While doing this I noticed that the linkset in `xxx.dbpedia.org/nl/test.rce.rnatoolset.net` seems wrong, as the `nl.dbpedia.org` is in the object rather then subject. Can someone verify if this is correct or if it needs to be fixed?

**Comment by kurzum** [comment]
needs to be fixed. 
`awk '{print $3,$2,$1 " ."}' MolensRijksmonument.nt`
does the job

**Comment by RPOD** [comment]
Changed it.
